bot/exts/moderation/modlog.py

```diff
@@ -729,6 +729,7 @@ async def on_raw_message_edit(self, event: discord.RawMessageUpdateEvent) -> Non
         """Log raw message edit event to message change log."""
         try:
             channel = self.bot.get_channel(int(event.data["channel_id"]))
+            await self.bot.wait_until_guild_available()
```

Comment:
This line needs to be at the start of the function, as it's the get_channel call that relies on the cache existing, which `wait_until_guild_available` guarantees

Reply:
should i add this to the top of most of the functions?

Reply:
This line would be best at the start of the function, yes. It needs to be before the get_channel call at least.

Reply:
ive added this to the top of the asynchronous functions that use the get_channel function call.

Reply:
Could you push the changes too?

Reply:
Lmfao I'm stupid I'll get it pushed now